### PR TITLE
README update

### LINKED ISSUE #9479: Grammar fixes
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [$FILENAME]($BRANCH_URL) in the `$REPO` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
- For more details about this gem, visit the official repository of whenever gem.
+ For more details about this gem, visit the official repository of the whenever gem.
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by OlaoluwaM**
I hope you don't mind, but I took the liberty to make some changes to wording that would make things more succinct and clearer. I also made certain items links because the way they were written kind of positioned them in a way that makes more sense to link them. Please let me know if there are any problems and I'll be sure to fix them. Thank you for this opportunity. I went through the docs and you guys are doing awesome things.

Fixes #9479  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Review by sahilsaha7773 — APPROVED** [review]
Great work @OlaoluwaM! Looks good to me, thanks for opening this PR!

**Comment by OlaoluwaM** [comment]
Great!! Thank you so much. So what do I do next 😅?

**Comment by sahilsaha7773** [comment]
> Great!! Thank you so much. So what do I do next 😅?

Please wait for the project maintainers to review the PR. 😄

**Review by NishantJawla — CHANGES_REQUESTED** [review]

File: README.md
Comment (OlaoluwaM) [reply README.md:14]: Here I treated the "what we call 'research notes'" as a sort of aside because its purpose is to add more information to/define what a "group research blog" is.  If we remove that clause the sentence would still make sense so the dashes were used to represent that its "extra" context/information. 

However, a comma could also work in place of the dashes as well. It's more or less the written equivalent of speaking to someone about something but quickly going off-topic to define or add more context to something you said before continuing with the main topic.

Comment (NishantJawla) [reply README.md:14]: Ok got it

**Review by NishantJawla — APPROVED** [review]

**Comment by NishantJawla** [comment]
> I hope you don't mind, but I took the liberty to make some changes to wording that would make things more succinct and clearer. I also made certain items links because the way they were written kind of positioned them in a way that makes more sense to link them. Please let me know if there are any problems and I'll be sure to fix them. Thank you for this opportunity. I went through the docs and you guys are doing awesome things.
> 
> Fixes #9479 (<=== Add issue number here)
> 
> Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!
> 
> * [ ✔] PR is descriptively titled 📑 and links the original issue above 🔗
> * [ ]  tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
> * [ ✔] code is in uniquely-named feature branch and has no merge conflicts 📁
> * [ ]  screenshots/GIFs are attached 📎 in case of UI updation
> * [ ]  ask `@publiclab/reviewers` for help, in a comment below
> 
> > We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!
> 
> If tests do fail, click on the red `X` to learn why by reading the logs.
> 
> Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
> 
> Thanks!

hey! @OlaoluwaM  please remove those "✔️" by editing. Save it then later tick when the box appears

**Comment by OlaoluwaM** [comment]
@NishantJawla Done!

**Comment by OlaoluwaM** [comment]
So us there anything else I need to do 🙂

**Comment by NishantJawla** [comment]
> So is there anything else I need to do 🙂

Just wait for a collaborator to review this PR

File: README.md
Comment (cesswairimu) [inline README.md:184]: I believe we would like to keep it as singular because the "reply by email comment" is one feature..what do you think?

File: README.md
Comment (cesswairimu) [reply README.md:14]: I think using colons would be better in this case. What so you think?

**Review by cesswairimu — CHANGES_REQUESTED** [review]
Tremendous!!!  thanks a lot @OlaoluwaM for going through the doc and helping improve it...please let me know what you think of my suggestions. Thanks again for working on this

File: README.md
Comment (NishantJawla) [reply README.md:14]: Yes, colons make a lot more sense in the current form, but he changed the sentence form so they can be removed.
Furthermore, using "what we call" in colons does not work well.

File: README.md
Comment (OlaoluwaM) [reply README.md:184]: Ohh, that makes a lot of sense. If that's the case wouldn't be better to string them together like "reply-by-email-to-comment" feature? Or put them in quotes so anyone reading knows that they go together

File: README.md
Comment (cesswairimu) [reply README.md:14]: oh yeah I see...cool lets leave it as is then. Thanks

File: README.md
Comment (cesswairimu) [reply README.md:184]: I agree, the quotes would make the phrase better. Thanks

Comment (OlaoluwaM) [reply README.md:184]: How do I update the PR?

Comment (cesswairimu) [reply README.md:184]: Update the changes from here https://github.com/OlaoluwaM/plots2/tree/patch-1 and commit, the commit will update over here. thanks

Comment (OlaoluwaM) [reply README.md]: Done!

**Review by cesswairimu — APPROVED** [review]
:tada:  thanks again  @OlaoluwaM

**Pull request merged**
